source/app.d

```diff
@@ -22,13 +23,16 @@ import vibe.d;
 Task s_checkTask;
 DubRegistry s_registry;
 DubRegistryWebFrontend s_web;
+string s_mirror;
 
 void startMonitoring()
 {
 	void monitorNewVersions()
 	{
+		sleep(2.minutes());
```

Comment:
I mostly added this as a throttling mechanism in case the process gets restarted repeatedly (maybe in case of a crash or manually during development). But you are right that this may not always be desired. Another reason is that adding code for cache warmup at application start is planned and that together with the update process would produce quite high load.

I don't know, maybe this can just be made configurable.